docs/topics/spiders.rst

```diff
@@ -294,6 +294,22 @@ The above example can also be written as follows::
         def start_requests(self):
             yield scrapy.Request(f'http://www.example.com/categories/{self.category}')
 
+
+In addition to that, you can also explicitly specify the spider attribute in ``crawl`` method of :class:`scrapy.crawler.CrawlerProcess` class.
+Then, they can also be used as :ref:`topics-feed-uri-params`::
+
+    from scrapy.crawler import CrawlerProcess
+    ...
+    
+    process = CrawlerProcess(settings={
+        "FEEDS": {
+            "%(category)s.json": {"format": "json"},
+        },
+    })
+    process.crawl(MySpider, category="electronics")
+
+See :ref:`run-from-script` for more information about :class:`scrapy.crawler.CrawlerProcess`.
```

Comment:
I would avoid the `FEEDS` example here, as that’s beyond the scope of this section. And I think we can make this part slightly shorter while providing the same information:

```suggestion
If you are :ref:`running Scrapy from a script <run-from-script>`, you can 
specify spider arguments when calling 
:class:`CrawlerProcess.crawl <scrapy.crawler.CrawlerProcess.crawl>` or
:class:`CrawlerRunner.crawl <scrapy.crawler.CrawlerRunner.crawl>`::

    process = CrawlerProcess()
    process.crawl(MySpider, category="electronics")
```

Reply:
LGTM!